readme.md

```diff
@@ -1,64 +1,72 @@
-# 📃 spotify.com-clone
+# 📃 Designing Our Own Grid-Based CSS Framework - Spotify.com Clone
 
-this project is part of a series of projects to be completed by students of [Microverse](https://www.microverse.org/ 'The Global School for Remote Software Developers!').
+This project is part of a series of projects to be completed by students of [Microverse](https://www.microverse.org/ 'The Global School for Remote Software Developers!').
 
-in this project, you’ll get a chance to try out twitter’s bootstrap framework by building a site that actually uses it – wewsweek.com.
+In this project, our brief was to create our own CSS Framework, following the project to recreate a webpage using Bootstrap.
 
 
+## 🔗 Link to assignment
 
-## 🔗 link to assignment
+The assignment can be found in the Odin Project's HTML and CSS course that can be found [here]https://www.theodinproject.com/courses/html5-and-css3/lessons/design-your-own-grid-based-framework).
 
-the assignment can be found in the Odin Project's HTML and CSS course that can be found [here]https://www.theodinproject.com/courses/html5-and-css3/lessons/design-your-own-grid-based-framework).
 
 
+## 📡 Technologies used
 
-## 📡 technologies used
+- HTML
+- CSS
+- Bootstrap
```

Comment:
You might want to reference the framework you created here instead of Bootstrap since this project is not using Bootstrap.

Reply:
hey, sorry for not adding out on framework as reference. 

readme.md updated! thanks